src/models/User.js

```diff
@@ -18,6 +18,21 @@ const userSchema = new Schema({
             message: 'Invalid email',
         },
     },
+    organizator: {
```

Comment:
I don't think it's the right code? First, you have a validation function inside if for a "Name" ? Second, required true?  Is this the "organizatorName" instead of the organizator itself ?

Reply:
fixed